ocs_ci/framework/__init__.py

```diff
@@ -146,6 +146,14 @@ def __init__(self):
         self.single_cluster_default = True
         self._single_cluster_init_cluster_configs()
 
+    @property
+    def default_cluster_ctx(self):
```

Comment:
Please add a docstring to this method. We are already missing docstrings for most of MultiClusterConfig (https://github.com/red-hat-storage/ocs-ci/issues/5831) but we shouldn't add more undocumented content.

Reply:
Okay, that will do.